paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodLayoutUITest.kt

```diff
@@ -47,6 +49,7 @@ internal class PaymentMethodLayoutUITest(
             displayedSavedPaymentMethod = PaymentMethodFixtures.displayableCard(),
             availableSavedPaymentMethodAction =
             PaymentMethodVerticalLayoutInteractor.SavedPaymentMethodAction.MANAGE_ALL,
+            mandate = null,
```

Comment:
I think it's time to move all of these into a factory. I'll follow up with the change to do so.